metrics/storage_collector.go

```diff
@@ -0,0 +1,85 @@
+package metrics
+
+import (
+	"context"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/rs/zerolog"
+)
+
+type StorageSizeCollector struct {
```

Comment:
we need an integration test for this

Reply:
Are you sure we want to invest time into it? As discussed previously, this metric will be replaced soon or later.